From ed019c1ee5f50e83eb5747827616d445cfbfd645 Mon Sep 17 00:00:00 2001
From: Christian Oder <myself5@carbonrom.org>
Date: Sun, 17 Feb 2019 21:23:59 +0100
Subject: [PATCH] sepolicy: define sysfs_dt_firmware_android as proc_type

We're using sysfs_dt_firmware_android for proc rules
on qcom/sepolicy-legacy and older Nexus devices (e.g. shamu)

Change-Id: I2041e4fc21bbec0e5104fb3795c23fd15b169c02
---
 public/file.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/public/file.te b/public/file.te
index ead5dfa6..d289a2a3 100644
--- a/public/file.te
+++ b/public/file.te
@@ -76,7 +76,7 @@ type sysfs_uio, sysfs_type, fs_type;
 type sysfs_batteryinfo, fs_type, sysfs_type;
 type sysfs_bluetooth_writable, fs_type, sysfs_type, mlstrustedobject;
 type sysfs_dm, fs_type, sysfs_type;
-type sysfs_dt_firmware_android, fs_type, sysfs_type;
+type sysfs_dt_firmware_android, fs_type, sysfs_type, proc_type;
 type sysfs_ipv4, fs_type, sysfs_type;
 type sysfs_kernel_notes, fs_type, sysfs_type, mlstrustedobject;
 type sysfs_leds, fs_type, sysfs_type;
-- 
2.17.1

